db/versions/0023.yml

```diff
@@ -0,0 +1,42 @@
+version: 23
+migrationScript: 0023-migration.sql
+downgradeScript: 0023-downgrade.sql
+methods:
+  add_github_accesstoken:
```

Comment:
The table puts a `_` in `access_token`, but this does not.  Any particular reason for the distinction?

Reply:
Ah quite right - I've embedded a `_` to make it consistent now.